src/applications/disability-benefits/all-claims/tests/pages/disabilityConditions/shared/sideOfBody.unit.spec.jsx

```diff
@@ -59,68 +59,43 @@ describe('526 side of body shared page', () => {
     expect(view.getByRole('button', { name: /submit/i })).to.exist;
   });
 
-  it('allows submit without a selection (not required)', async () => {
+  it('blocks submit without a selection (required)', async () => {
     const onSubmit = sinon.spy();
-    const { container } = mountPage({}, onSubmit);
-    const view = within(container);
+    const { container, getByRole } = mountPage({}, onSubmit);
 
-    view.getByRole('button', { name: /submit/i }).click();
+    fireEvent.click(getByRole('button', { name: /submit/i }));
 
     await waitFor(() => {
-      expect(onSubmit.calledOnce).to.be.true;
+      const radio = container.querySelector('va-radio');
+      expect(radio).to.have.attribute('error', 'You must provide a response');
     });
+
+    expect(onSubmit.called).to.be.false;
```

Comment:
changed to .called because we don't care about an exact count. Before, we did because we wanted to know the submit button was clicked only once as that means they were able to proceed to the next step without issue. Now, we are checking the false case, where no matter how many times they click submit they shouldn't be able to move to the next step